SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -31,5 +32,19 @@ public async Task<ActionResult<ArticleResponseDTO>> GetArticleByTitleAsync(strin
             var articleResponse = _mapper.Map<ArticleResponseDTO>(article);
             return Ok(articleResponse);
         }
+
+        [Authorize(Roles = "Admin")]
+        [HttpPost]
+        public async Task<IActionResult> CreateArticleAsync([FromBody] CreateArticleDTO adminInput)
+        {
```

Comment:
Add validation for the input fields, and check restrictions in the database and in GitHub acceptance. Add a Fluent validator for the input restrictions, and check uniqueness in service layer